core/trino-main/src/main/java/io/trino/sql/planner/plan/TableFunctionProcessorNode.java

```diff
@@ -211,6 +268,14 @@ public List<Symbol> getOutputSymbols()
         return symbols.build();
     }
 
+    @Nullable
```

Comment:
there were 2 constructors, one assumed this will never be null and second just it to null, i will change it so there is one constructor and the field can be null